source/ceylon/test/engine/internal/file.ceylon

```diff
@@ -0,0 +1,39 @@
+import java.io {
+    File,
+    FileOutputStream,
+    PrintWriter
+}
+
+native class FileFacade(String path) satisfies Destroyable {
```

Comment:
The JS version of your facade doesn’t seem to be working, even without my change… I’m just getting an empty file. Did you test it when you added it?